libs/@hashintel/block-design-system/src/icons/caret-down.tsx

```diff
@@ -10,7 +13,10 @@ export const CaretDownIcon: FunctionComponent<SvgIconProps> = (props) => {
       fill="none"
       {...props}
     >
-      <path d="M137.4 374.6c12.5 12.5 32.8 12.5 45.3 0l128-128c9.2-9.2 11.9-22.9 6.9-34.9s-16.6-19.8-29.6-19.8L32 192c-12.9 0-24.6 7.8-29.6 19.8s-2.2 25.7 6.9 34.9l128 128z" />
+      <path
+        fill={fill}
```

Comment:
Thank you, I've fixed this in https://github.com/hashintel/hash/pull/2884/commits/9ad916604bfb4847e144d36a7b8dc94f7f297adc

It wasn't working because I was passing `fill` directly to `CaretDownIcon` instead of inside `sx`. Moving it into `sx` fixed it